tests/conftest.py

```diff
@@ -201,6 +201,13 @@ async def guard_func(*args, **kwargs):
 util.get_local_ip = lambda: "127.0.0.1"
 
 
+@pytest.fixture(name="caplog")
+def caplog_fixture(caplog):
+    """Test that capture logs are likely needing debug level."""
```

Comment:
If the idea is to keep this fixture, we should improve the docstring, maybe:
```suggestion
    """Set log level to debug for tests using the caplog fixture."""
```